Mathlib/Analysis/LocallyConvex/WithSeminorms.lean

```diff
@@ -418,7 +418,7 @@ theorem SeminormFamily.withSeminorms_iff_topologicalSpace_eq_iInf [TopologicalAd
       t = ⨅ i, (p i).toSeminormedAddCommGroup.toUniformSpace.toTopologicalSpace := by
   rw [p.withSeminorms_iff_nhds_eq_iInf,
     TopologicalAddGroup.ext_iff inferInstance (topologicalAddGroup_iInf fun i => inferInstance),
-    nhds_iInf]
+    _root_.nhds_iInf]
```

Comment:
Why is this necessary now? If it isn't please remove it.

Reply:
If I remove it, I get the error "ambiguous, possible interpretations" between `_root_` and `Filter`